### PR TITLE
USB HS support fixes on STM32U5

### PR DESCRIPTION
**Pull request by erikarn**
this is a WIP (i haven't even checkpatched it!) to address Issues #79823 and Issue #79825

### CODE REVIEW

**Comment by jfischer-no** [comment]
Can you please familiarize yourself with how to write commit titles and messages, and the PR submission process? Read the link posted by the bot above. Use `git log` to get an idea of how to write commit messages.

File: samples/subsys/usb/cdc_acm/nucleo_u5a5zj_q.overlay [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
```
Comment (jfischer-no) [inline samples/subsys/usb/cdc_acm/nucleo_u5a5zj_q.overlay:1]: No overlays.

**Review by jfischer-no — CHANGES_REQUESTED** [review]

**Comment by erikarn** [comment]
> Can you please familiarize yourself with how to write commit titles and messages, and the PR submission process? Read the link posted by the bot above. Use `git log` to get an idea of how to write commit messages.

oh absolutely, I'm slowly massaging this into the right shape. I just wanted to first make sure I got the code somewhere in case I got busy!

**Comment by kartben** [comment]
> > Can you please familiarize yourself with how to write commit titles and messages, and the PR submission process? Read the link posted by the bot above. Use `git log` to get an idea of how to write commit messages.
> 
> oh absolutely, I'm slowly massaging this into the right shape. I just wanted to first make sure I got the code somewhere in case I got busy!

You might want to set the PR as draft in the meantime so that reviewers know it's not ready for review just yet (but you might still get early feedback and, to your point, this would have the merit of putting the code somewhere :))

**Comment by erikarn** [comment]
Ok! Lemme go find how to mark it as draft, I've not done that before! heh

**Pull request closed without merging**